src/decorator.ts

```diff
@@ -66,13 +66,41 @@ export interface EnumValueMetadata {
 
 export interface FieldOption {
     type?: any;
+    description?: string;
+    nonNull?: boolean;
 }
 
 export interface ArgumentOption {
     name: string;
     type?: any;
 }
 
+export interface ListOption {
```

Comment:
Since all these `description` fileds mean the same thing and it is part of every available option object, what if you'd extract them to an interface and then you'd extend that interface from all those options?  

Something like:
```typescript
export interface DefaultOption {
  description?: string;
}

export interface ObjectTypeOption extends DefaultOption { }
```

Reply:
Yes I thought about that also but kept it for next step, I was thinking about refactoring current code base, for now everything (about decorators) is in one file and maybe it would be good to split it in separate files, file per decorator + 1 common file.